### PR TITLE
MediaKeysCapabilitiesTest Client

### PR DESCRIPTION
**Pull request by skyshaha1**
Summary: Test the MediaKeysCapabilities APIs in the client component tests.
Type: Feature
Test Plan: Unittests, Component Tests.
Jira: RIALTO-439

### CODE REVIEW

File: tests/componenttests/client/CMakeLists.txt [hunk 1]
```diff
@@ -102,6 +103,7 @@ target_include_directories(
         $<TARGET_PROPERTY:RialtoPlayerPublicClientMocks,INTERFACE_INCLUDE_DIRECTORIES>
         $<TARGET_PROPERTY:RialtoTestCommonMatchers,INTERFACE_INCLUDE_DIRECTORIES>
         $<TARGET_PROPERTY:RialtoTestCommonProtoUtils,INTERFACE_INCLUDE_DIRECTORIES>
+        $<TARGET_PROPERTY:RialtoServerMain,INCLUDE_DIRECTORIES>
```
Comment (skywilliamsonl) [inline tests/componenttests/client/CMakeLists.txt:106]: Move the keysystems to playercommon so we dont have to include server side main code.

**Review by skywilliamsonl — DISMISSED** [review]

File: tests/componenttests/client/tests/eme/MediaKeysCapabilitiesTest.cpp [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+ *   Check if key is supported
+ *
+ *  Step 4: Check if key is supported - failure
+ *   supportsKeySystem.
+ *   Expect that supportsKeySystem is propagated to server.
+ *   Api call returns with failure.
+ *
+ *  Step 5: Get the supported key system version - success
+ *   getSupportedKeySystemVersion.
+ *   Expect that supportsKeySystem is propagated to server.
```
Comment (skyproudmanp) [inline tests/componenttests/client/tests/eme/MediaKeysCapabilitiesTest.cpp:72]: getSupportedKeySystemVersion not supportedKeySystem is propagated

File: tests/componenttests/client/tests/base/MediaKeysTestMethods.h [hunk 3]
```diff
@@ -89,6 +94,15 @@ class MediaKeysTestMethods
     void shouldNotifyKeyStatusesChanged();
     void shouldNotifyLicenseRenewal();
 
+    // MediaKeysCapabilities Expect methods
+    void createMediaKeysCapabilitiesObject();
```
Comment (skyproudmanp) [inline tests/componenttests/client/tests/base/MediaKeysTestMethods.h:98]: create & destroy are API methods not expect methods

File: tests/componenttests/client/tests/eme/MediaKeysCapabilitiesTest.cpp [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+ *   Api call returns with success.
+ *   Check if version is supported for the key system
+ *
+ *  Step 6: Get the supported key system version - failure
+ *   getSupportedKeySystemVersion.
+ *   Expect that supportsKeySystem is propagated to server.
```
Comment (skyproudmanp) [inline tests/componenttests/client/tests/eme/MediaKeysCapabilitiesTest.cpp:78]: getSupportedKeySystemVersion not supportedKeySystem is propagated

**Review by skywilliamsonl — APPROVED** [review]

**Pull request merged**